lib/conjur/fpm/debify_utils.sh

```diff
@@ -9,3 +9,24 @@ function bundle_clean() {
     rm -rf vendor/bundle/ruby/${ruby_version}/gems/*/{test,spec,examples,example,contrib,doc,ext,sample}
   fi
 }
+
+# Remove files from the current directory that also exist in another given
+# directory. For example, say in the current directory there is:
+#    foo
+#    bar/baz
+#    bar/xyzzy
+#    bacon
+#    people/phlebas
+# and in dir2 there is
+#    bacon
+#    alice
+#    people/phlebas
+#    bar/xyzzy
+# then after running `remove_matching dir2` current directory will be left with only:
+#    foo
+#    bar/baz
+# Note it probably isn't 100% fool-proof, so don't launch it out to space or something.
```

Comment:
🤣

Reply:
Or give a lone `.` as the argument, for that matter ;)

Reply:
Actually, nevermind, this should work I suppose. I've had a thinko that this would remove stuff from the root dir, but with the sed script there should always be a dot at the beginning, so nevermind ;)